### PR TITLE
Update changelog and docs version for release v1.4.0

### PR DESCRIPTION
**Pull request by jetuk**

### CODE REVIEW

**Comment by jetuk** [comment]
It would be nice to get #795 and #774 merged before this release.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,24 @@
...
+## v1.4.0
+
+### New Features
+
+- Added support time-steps based on Pandas offsets (#675)
+- Added `InterpolatedFlowParameter` (#740)
+- Added support for `percentile` and `percentileofscore` aggregation functions (#777)
+- Added `PiecewiseIntegralParameter` (#772)
+- Added support for including references to Python modules in JSON format (#765)
+- Added `CurrentYearThresholdParameter` and `CurrentOrdinalDayThresholdParameter` parameters (#789)
```
Comment (Batch21) [inline CHANGELOG.md]: Also add these two params to the api docs?

Comment (jetuk) [reply CHANGELOG.md]: #799 I added this issue. None of the threshold ones are there, so I didn't think it should be done here.

Comment (Batch21) [reply CHANGELOG.md]: ah ok, I missed that

**Pull request merged**